Shell/SALShell/SALShell/SymbolTable/Scope.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SALShell.SymbolTable
+{
+    class Scope
+    {
+        public Scope(Scope parent, string scopeName, int Depth)
+        {
+            Parent = parent;
+            this.scopeName = scopeName;
```

Comment:
this. bruger du til at differentierer mellem constructor param og dit field fordi de har samme navn, men så gør du ikke det ved parent? burde man ikke stick til én convention (undskyld søren, ved ikke hvad jeg ellers skal finde)